www/src/data/sidebars/doc-links.yaml

```
@@ -469,6 +469,8 @@
                   link: /docs/processing-payments-with-stripe/
                 - title: Processing Payments with Square
                   link: /docs/processing-payments-with-square/
+                - title: Working with Etsy in Gatsby
+                  link: /docs/working-with-Etsy-in-Gatsby/
```

This isn't actually hurting anything and may only be personal preference but I'd keep filenames lowercase for consistency. That would mean a change here and renaming the Markdown file itself.

```suggestion
                  link: /docs/working-with-etsy-in-gatsby/
```